### PR TITLE
Support custom URL for static assets

### LINKED ISSUE #297: Improve publicPath
### What's up?

`publicPath` currently maps a static dir to the `/loader/` path (which is `/` as far as the user bundle is concerned). This works for some, but a common pattern is to map the static folder to a path other than `/`, like `/static/`), which means `publicPath` should maybe become a dict, like:

```js
publicPath: {
  '/static/': '/path/to/static/files/'
}
```

### Mkay, tell me more...

This is going to make sense after #284 is merged. Exporting needs to be taken into account fully in this issue.

TODO: Provide more context

### PR DESCRIPTION
**Pull request by skidding**
Closes #297

Also on this branch:
- [x] Export static assets as well
- [x] Relative paths inside CP #397
- [x] Fix CP UI in Firefox
- [x] ~~Fix CP in Safari~~ It's not perfect (fixture editor sometimes overlaps with Loader frame), but it's outside of scope and *much* better than before

### CODE REVIEW

**Comment by skidding** [comment]
@jlc467 bundled a fix for #397 as well, let me know if this is what you also had in mind

File: packages/react-cosmos-component-playground/src/components/ComponentPlayground/index.jsx [hunk 1]
```diff
@@ -319,7 +319,7 @@ export default class ComponentPlayground extends Component {
                 <a
                   ref="fixtureEditorButton"
                   className={fixtureEditorClassNames}
-                  href={`/${fixtureEditorUrl}`}
+                  href={`${fixtureEditorUrl}`}
```
Comment (ovidiubute) [inline packages/react-cosmos-component-playground/src/components/ComponentPlayground/index.jsx]: How about `href={fixtureEditorUrl}`

File: packages/react-cosmos-component-playground/src/components/ComponentPlayground/index.jsx [hunk 2]
```diff
@@ -328,7 +328,7 @@ export default class ComponentPlayground extends Component {
                 <a
                   ref="fullScreenButton"
                   className={styles.button}
-                  href={`/${fullScreenUrl}`}
+                  href={`${fullScreenUrl}`}
```
Comment (ovidiubute) [inline packages/react-cosmos-component-playground/src/components/ComponentPlayground/index.jsx]: `href={fullScreenUrl}`

File: packages/react-cosmos-component-playground/src/components/FixtureList/index.jsx [hunk 3]
```diff
@@ -120,7 +122,7 @@ export default class FixtureList extends Component {
                       <a
                         key={j}
                         className={fixtureClassNames}
-                        href={`/${uri.stringifyParams(nextUrlParams)}`}
+                        href={`${uri.stringifyParams(nextUrlParams)}`}
```
Comment (ovidiubute) [inline packages/react-cosmos-component-playground/src/components/FixtureList/index.jsx]: Unnecessary String interpolation.

File: packages/react-cosmos-component-playground/src/components/ComponentPlayground/index.jsx (hunk 2 above)
Comment (skidding) [inline packages/react-cosmos-component-playground/src/components/ComponentPlayground/index.jsx]: 😆 I just removed the slash without thinking 🐑🐑🐑 

Thanks!

File: packages/react-cosmos-component-playground/src/components/FixtureList/index.jsx (hunk 3 above)
Comment (skidding) [inline packages/react-cosmos-component-playground/src/components/FixtureList/index.jsx]: Fixed!

**Pull request merged**